.github/workflows/api_file_update.yaml

```diff
@@ -17,10 +17,12 @@ jobs:
         if: github.event_name == 'pull_request' && github.event.pull_request.merged == true
       - name: Update API file
         run: |
-          cd specs
-          wget https://eaterslab.herokuapp.com/api.yaml -O api_update.yaml; then
-          rm -rf api.yaml
-          mv api_update.yaml api.yaml
+          cd ./specs
+          if wget https://eaterslab.herokuapp.com/api.yaml -O api_update.yaml; then
+            rm -rf api.yaml
```

Comment:
Do we need `&&` piping in here? Can unfinished changes propagate to PR stage?

Reply:
When api.yaml not changes then the PR is not created because there are no changes in the branch